Subject: webkit-gtk not found, use webkitgtk instead

Upstream-Status: Private

Signed-off-by: Prajankya Sonar <contact@prajankya.me>

diff --git recipes-extended/wxwidgets/wxwidgets_2.9.5.bb recipes-extended/wxwidgets/wxwidgets_2.9.5.bb
index 1996556..3d29b9c 100644
--- recipes-extended/wxwidgets/wxwidgets_2.9.5.bb
+++ recipes-extended/wxwidgets/wxwidgets_2.9.5.bb
@@ -4,7 +4,7 @@ HOMEPAGE = "http://www.wxwidgets.org"
 LICENSE = "WXwindows"
 LIC_FILES_CHKSUM = "file://docs/licence.txt;md5=18346072db6eb834b6edbd2cdc4f109b"
 
-DEPENDS = "webkit-gtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
+DEPENDS = "webkitgtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
 
 SRC_URI = "${SOURCEFORGE_MIRROR}/wxwindows/wxWidgets-${PV}.tar.bz2"
 SRC_URI[md5sum] = "e98c5f92805493f150656403ffef3bb0"
